src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtils.java

```diff
@@ -371,6 +410,147 @@ private static Properties readConfigFileProperties(final Path configFilePath) {
+                            versionMinor  = Integer.valueOf(m.group(2));
+                            versionYear   = Integer.valueOf(m.group(3));
+                            versionMonth  = Integer.valueOf(m.group(4));
+                            versionDay    = Integer.valueOf(m.group(5));
+
+                            version = versionMajor + "." + versionMinor + "." + versionYear + "" + versionMonth + "" + versionDay;
+                        }
+                        else {
+                            logger.warn("README file has improperly formatted version string: " + line);
+                        }
+                    }
+
+                    if (source == null && line.startsWith(README_SOURCE_LINE_START)) {
+                        final Matcher m = SOURCE_PATTERN.matcher(line);
+                        if ( m.matches() ) {
+                            source = m.group(1);
+                        }
+                        else {
+                            logger.warn("README file has improperly formatted source string: " + line);
+                        }
+                    }
+
+                    if (alternateSource == null && line.startsWith(README_ALT_SOURCE_LINE_START)) {
+                        final Matcher m = ALT_SOURCE_PATTERN.matcher(line);
+                        if ( m.matches() ) {
+                            alternateSource = m.group(1);
+                        }
+                        else {
+                            logger.warn("README file has improperly formatted alternate source string: " + line);
+                        }
+                    }
+
+                    line = reader.readLine();
+                }
+
+                // Make sure we have good info:
+                if ( version == null ) {
+                    logger.warn("Unable to read version information from data sources info/readme file: " + readmePath.toUri().toString());
+                }
+                else {
+                    logger.info("Data sources version: " + version);
+
+                    // Make sure our version is OK:
+                    dataSourcesPathIsAcceptable = validateVersionInformation(versionMajor, versionMinor, versionYear, versionMonth, versionDay);
+                }
+
+                if ( source == null ) {
+                    logger.warn("Unable to read source information from data sources info/readme file: " + readmePath.toUri().toString());
+                }
+                else {
+                    logger.info("Data sources source: " + source);
+                }
+
+                if ( alternateSource == null ) {
+                    logger.warn("Unable to read alternate source information from data sources info/readme file: " + readmePath.toUri().toString());
+                }
+                else {
+                    logger.info("Data sources alternate source: " + alternateSource);
+                }
+            }
+            catch (final Exception ex) {
```

Comment:
Please log information about the exception.  Generally, you should not be catching `Exception`, but rather a subclass of `Exception`.

Reply:
Sounds good.

I want to catch any exceptions that happen there and continue so that it doesn't actually fail.  This is an optional check for logging purposes anyway.